backend/otr/src/test/java/fi/oph/otr/util/QualificationUtilTest.java

```diff
@@ -0,0 +1,79 @@
+package fi.oph.otr.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import fi.oph.otr.api.dto.clerk.ClerkQualificationDTO;
+import fi.oph.otr.model.QualificationExaminationType;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+
+public class QualificationUtilTest {
+
+  // Contains duplicate FI-SV lang pair
```

Comment:
Mikäs tämän FI-SV-duplikaatin idea oikein olikaan?

Reply:
Kuvastaa sitä, että nykyisellä toteutuksella mikäli tulkilla on kaksi voimassaolevaa rekisteröintiä samalle kieliparille, palauttaa backend näistä molemmat ja virkailija näkee näin listausnäkymässä molemmat ko. rekisteröinnit. De-duplikointi siis toimii vain umpeutuneille rekisteröinneille. AKR:ssä pätee sama auktorisointien osalta.